clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/api/KubernetesClientApiConverter.groovy

```diff
@@ -349,8 +356,8 @@ class KubernetesClientApiConverter {
     def persistentVolumeClaims = new ArrayList<V1PersistentVolumeClaim>()
     if (description.volumeClaims) {
       description.volumeClaims.forEach({ claim ->
-        V1PersistentVolumeClaimSpec spec = new V1PersistentVolumeClaimSpec()
-        V1ObjectMeta metadata = new V1ObjectMeta()
+        def spec = new V1PersistentVolumeClaimSpec()
```

Comment:
Why replace the type with `def`?

Reply:
No particular reasons.  I just think that the code already initialized with the type.